### PR TITLE
Refactoring autelis test; Improve stability for slow environments

### PR DESCRIPTION
**Pull request by IgorProtsenkoOd**

### CODE REVIEW

File: src/e2e-tests/page-objects/autelis.page.js [hunk 1]
```diff
@@ -7,200 +7,182 @@ let counter = 0;
 class AutelisPage extends Page {
     get pageContent() {return $('//div[contains(@class, "ion-page")]')}
     get autelisTab() {return $('#autelis-tabs-tab-autelis')}
-    get offButton() {return $('//button[text()[contains(., "OFF")]]')}
-    get poolButton() {return $('//button[text()[contains(., "POOL")]]')}
-    get spaButton() {return $('//button[text()[contains(., "SPA")]]')}
+    get allOffButton() {return $('//ion-segment-button[text()[contains(., "OFF")]]')}
```
Comment (popescunsergiu) [inline src/e2e-tests/page-objects/autelis.page.js]: Off buttons can be split as they were. no need for alloffbuttons and indexes

File: src/e2e-tests/page-objects/autelis.page.js [hunk 2]
```diff
@@ -7,200 +7,182 @@ let counter = 0;
...
-    // 0 - lower; 1 - higher
-    get spaHeatAdjButtons() {return $$('//div[text()="Spa Heat"]/parent::*//div[@class="float-right"]/button')}
-    get spaHeatCounter() {return $('//div[text()="Spa Heat"]/parent::*//div[@class="float-right"]/input')}
+    get switchMessagePool() {return $('//*[contains(@class, "AutelisTab_mainSwitch")]/parent::*//div[text()[starts-with(., "Pool")]]')}
+    get switchMessageSpa() {return $('//*[contains(@class, "AutelisTab_mainSwitch")]/parent::*//div[text()[starts-with(., "Spa")]]')}
+    get switchMessageSolar() {return $('*//div[text()[starts-with(., "Solar")]]')}
+
+    // 0 - Solar; 1 - Cleaner; 2 - Pool Heat; 3 - Pool Light; 4 - Waterfall; 5 - Spa Heat;
+    // 6 - Jets; 7 - Spa Light; 8 - Blower
+    get onButton() {return $$('//*[contains(@class, "AutelisTab_toggle")]//ion-label[text()="On"]')}
```
Comment (popescunsergiu) [inline src/e2e-tests/page-objects/autelis.page.js]: Please keep on and off buttons as they were, section based. 
Here is an example of an xPath that works for `cleanerOffButton`:
`//div[text()="Cleaner"]/parent::*//ion-segment-button[.="On"]`

Please note the difference between `text()` and `.` in xPath

Comment (IgorProtsenkoOd) [reply src/e2e-tests/page-objects/autelis.page.js]: Changed.

Comment (IgorProtsenkoOd) [reply src/e2e-tests/page-objects/autelis.page.js]: Changed.

**Pull request merged**